github/Authorization.py

```diff
@@ -43,21 +45,22 @@ class Authorization(github.GithubObject.CompletableGithubObject):
     This class represents Authorizations. The reference can be found here https://docs.github.com/en/enterprise-server@3.0/rest/reference/oauth-authorizations
     """
 
-    _app: Attribute["AuthorizationApplication"]
-    _created_at: Attribute[datetime]
-    _id: Attribute[int]
-    _note: Attribute[Optional[str]]
-    _note_url: Attribute[Optional[str]]
-    _scopes: Attribute[str]
-    _token: Attribute[str]
-    _updated_at: Attribute[datetime]
-    _url: Attribute[str]
```

Comment:
I actually like these explicit field declarations (I'm a Java/kotlin developer primarily)

Reply:
declare attribute in `def _initAttributes(self):` so we don't need to write these twice and keep them syncing